App/static/css/estilos.css

```
@@ -0,0 +1,58 @@
+body {
+  background-color: rgba(185, 196, 185, 0.2);
+}
+
+.calc-container {
+  display: block;
+}
+
+h2 {
+  text-align: center;
+}
+
+.calc {
+  margin: 0 auto;
+  padding: 20px;
+  background-color: rgba(63, 80, 241, 0.4);
+  width: 300px;
+  height: 300px;
+  border-radius: 10px;
+  padding: 20px;
+  user-select: none;
```

add prefix for internet explorer, and mozila support
```suggestion
  user-select: none;
  
    -webkit-touch-callout: none; /* iOS Safari */
    -webkit-user-select: none; /* Safari */
     -khtml-user-select: none; /* Konqueror HTML */
       -moz-user-select: none; /* Old versions of Firefox */
        -ms-user-select: none; /* Internet Explorer/Edge */
            user-select: none; /* Non-prefixed version, currently
                                  supported by Chrome, Edge, Opera and Firefox */
```

---

fixed